include/wlr/interfaces/wlr_data_source.h

```diff
@@ -1,16 +0,0 @@
-#ifndef WLR_INTERFACES_WLR_DATA_SOURCE_H
-#define WLR_INTERFACES_WLR_DATA_SOURCE_H
-
-#include <wlr/types/wlr_data_source.h>
-
-struct wlr_data_source_impl {
-	void (*send)(struct wlr_data_source *data_source, const char *type, int fd);
-	void (*accepted)(struct wlr_data_source *data_source, const char *type);
-	void (*cancelled)(struct wlr_data_source *data_source);
-};
```

Comment:
This was there to allow non-wayland data sources on compositor side. Would be needed for x11 <-> wayland clipboard synchronization and supporting stuff like the sway set_clipboard command (or whatever a compositor wants to do with the clipboard).

Reply:
Oh I see thanks.